frontend/src/scenes/data-warehouse/editor/OutputPane.tsx

```diff
@@ -0,0 +1,421 @@
+import 'react-data-grid/lib/styles.css'
```

Comment:
I get that we're just dropping this in from existing code. Still, we need to replace the current editor ASAP - having multiple versions of this code will cause nightmares when people update existing BI stuff without knowing this exists.